CoastalResilienceNetwork/GeositeFramework
# Use json file extension for translation files

## Overview

The previous implementation used a dynamic endpoint to serve the translation file, which didn't have a file extension. When we initially migrated the translation backend to the static build, we left the file without an extension. This worked for most installations, but proved to be problematic when serving the app from IIS, which uses a whitelist for serving file extensions. Adding the json file extension also makes more practical sense than using no extension since the file is indeed JSON.

Connects #1163 

## Testing Instructions

- Build the `dr-region` site on the `feature/task-1` branch. 
- Deploy the region as virtual directory to IIS. I had success doing this by creating a website that had a root at `C:\inetpub\wwwroot\tnc\` and then adding a virtual directory under that site at `C:\inetpub\wwwroot\tnc\dr`.
- Access the site, and verify the request to return the `es` file fails.
- Rebuild the site on this branch.
- Deploy it to IIS in the same way as above.
- Access the site, and verify the request returns the `es.json` file succesfully.
